app/views/idv/index.html.slim

```diff
@@ -3,42 +3,36 @@
 h1.h3.my0 = t('idv.titles.expectations')
 
 .mb-40p
-  .mb-40p
-    | To complete registration with login.gov, we’ll ask you about the following:
-  h2.mb1.pb-tiny.h5.border-bottom.border-teal Basic information
-  .mb-12p.h6 We’ll start with basic information, like:
+  .mb-40p = t('idv.index.subheader')
+  h2.mb1.pb-tiny.h5.border-bottom.border-teal = t('idv.index.section_1.header')
+  .mb-12p.h6 = t('idv.index.section_1.subheader')
   ul.mb0.teal-dots
-    li.mb1 Full name
-    li.mb1 Date of birth
-    li.mb1 Mailing address
-    li.mb1 Social Security Number or Individual Tax Identification Number
+    li.mb1 = t('idv.index.section_1.bullet_1')
+    li.mb1 = t('idv.index.section_1.bullet_2')
+    li.mb1 = t('idv.index.section_1.bullet_3')
+    li.mb1 = t('idv.index.section_1.bullet_4_html')
 
 .mb-40p
-  h2.mt0.mb1.pb-tiny.h5.border-bottom.border-teal Financial information
-  .mb-12p.h6
-    | Additionally, we need a number from only one of your financial accounts.
-      We can use any of the following:
+  h2.mb1.pb-tiny.h5.border-bottom.border-teal = t('idv.index.section_2.header')
+  .mb-12p.h6 = t('idv.index.section_2.subheader')
   ul.mb-12p.teal-dots
-    li.mb1 Last 8 digits of a credit card
-    li.mb1 Auto loan account number
-    li.mb1 Mortgage loan account number
-    li.mb1 Home equity line of credit account number
-  .h6.italic
-    | You will never be charged any money, and are not sharing any account balances,
-      or other financial information with us.
+    li.mb1 = t('idv.index.section_2.bullet_1')
+    li.mb1 = t('idv.index.section_2.bullet_2')
+    li.mb1 = t('idv.index.section_2.bullet_3')
+    li.mb1 = t('idv.index.section_2.bullet_4')
+  .h6.italic = t('idv.index.section_2.footnote')
 
 .mb-40p
-  h2.mt0.mb1.pb-tiny.h5.border-bottom.border-teal Official phone number
-  .mb-12p.h6 Finally, we need your official phone number, which can be:
+  h2.mb1.pb-tiny.h5.border-bottom.border-teal = t('idv.index.section_3.header')
+  .mb-12p.h6 = t('idv.index.section_3.subheader')
   ul.mb0.teal-dots
-    li.mb1 Lorem ipsum dolor set
-    li.mb1 Lorem ipsum dolor
+    li.mb1 = t('idv.index.section_3.bullet_1')
+    li.mb1 = t('idv.index.section_3.bullet_2_html')
+    li.mb1 = t('idv.index.section_3.bullet_3_html')
 
-p.mb3
-  | We have partnered with a trusted third party company to verify this information
-    and your identity.
+p.mb3 = t('idv.index.paragraph_1')
 
-p.mb3.bold Do you have all of this information available?
+p.mb3.bold = t('idv.index.prompt')
 
-= link_to 'Yes', idv_session_path, class: 'btn btn-primary btn-wide'
-.mt1 = link_to t('idv.messages.cancel_link'), idv_cancel_path, class: 'underline'
+= link_to t('idv.index.continue_link'), idv_session_path, class: 'btn btn-primary btn-wide'
+.mt1 = link_to t('idv.index.cancel_link'), idv_cancel_path, class: 'underline'
```

Comment:
This looks great! I18n supports relative keys, and every key in `idv.index` can be access without referencing `idv.index`. See: https://github.com/18F/identity-idp/blob/master/app/views/profile/index.html.slim#L16